### DISCUSSION TITLE
Enforce event registration with SSO accounts for certain events

### DISCUSSION THREAD

**Discussion by MacLake** [pretix/pretix#4697] [Ideas]
We have Single Sign ON (SSO, OpenID Connect) enabled with Pretix as the SSO client. 

We would like to restrict event registration to participants with SSO accounts for some, but not for all events. It seems that there is no way to limit event registration to SSO and disable event registration as a guest. The reason for this wish is that we offer courses where a customer relationship is necessary that goes far beyond events like a concert or festival.

P.S.: I have already written this, but I must have missed to send this proposal, at least I can’t find it.

**Comment by raphaelm** [comment]
There are some related discussions in #2991 and #3202
